frontends/web/src/components/devices/bitbox02/bitbox02.tsx

```diff
@@ -349,9 +362,10 @@ class BitBox02 extends Component<Props, State> {
                                     </div>
                                     <div className={style.buttons}>
                                         <button
+                                            disabled={!sdCardInserted}
                                             className={[style.button, style.primary].join(' ')}
                                             onClick={this.restoreBackup}>
-                                            Name Device & Continue
+                                            {sdCardInserted ? 'Name Device & Continue' : 'Insert SD Card with Backup'}
```

Comment:
looks hard to reason about. Better to make a new `<Step ...>` at the beginning to enter the SD card, as it needs to be inserted in any case.

Reply:
Included in a higher step